tests/foreman/longrun/test_inc_updates.py

```diff
@@ -147,17 +141,18 @@ def host(
     module_target_sat,
 ):
     # Create client machine and register it to satellite with rhel_7_partial_ak
-    rhel7_contenthost_module.install_katello_ca(module_target_sat)
+    # rhel7_contenthost_module.install_katello_ca(module_target_sat)
     # Register, enable tools repo and install katello-host-tools.
-    rhel7_contenthost_module.register_contenthost(
-        module_entitlement_manifest_org.label, module_ak.name
+    rhel7_contenthost_module.register(
+        module_sca_manifest_org, None, module_ak.name, module_target_sat
     )
-    rhel7_contenthost_module.enable_repo(REPOS['rhst7']['id'])
+    rhel7_contenthost_module.enable_repo(REPOS['rhsclient7']['id'])
     rhel7_contenthost_module.install_katello_host_tools()
```

Comment:
IMHO we don't need to install `katello_host_tools` because we don't need to execute `katello-package-upload` anymore after package installation because package profile upload is now handled by `subscription-manager repos` command.

Reply:
Sure, removing these 2 lines
`rhel7_contenthost_module.install_katello_host_tools()` and `rhel7_contenthost_module.execute('katello-package-upload')`